source/runtime.cpp

```diff
@@ -1081,6 +1093,23 @@ namespace reshade
 		if (!success)
 		{
 			LOG(ERROR) << "Failed to write screenshot to " << path << "!";
+			return;
+		}
+
+		if (_screenshot_include_preset)
+		{
+			if (_current_preset >= 0)
```

Comment:
This can be combined into a single branch `_screenshot_include_preset && _current_preset >= 0`.